src/bug_report.rs

```diff
@@ -99,12 +89,16 @@ fn make_github_issue_link(starship_version: &str, environment: Environment) -> S
         os_version = environment.os_version,
         shell_config = environment.shell_info.config,
         starship_config = environment.starship_config,
-    ));
+    ))
+        .replace("%20", "+");
 
     format!(
-        "https://github.com/starship/starship/issues/new?template={}&body={}",
```

Comment:
I believe the template query param here is required for the bug label to apply to the new issue.